app/questionnaire/rules/rule_evaluator.py

```diff
@@ -15,7 +16,7 @@
     ValueSourceTypes,
 )
 
-RuleEvaluatorTypes = Union[bool, Optional[date], list[str], list[date]]
+RuleEvaluatorTypes = Union[bool, Optional[date], list[str], list[date], int, Decimal]
```

Comment:
I would add floats as that could be possible from static value or metadata etc

Reply:
Added float 👍